components/Input.js

```diff
@@ -1,119 +1,66 @@
-import {
-  TextInput,
-  TextSize,
-  Color,
-  Inline,
-  Stack,
-  Text,
-  TextWeight,
-  TextLeading,
-} from "@ableco/baseline";
+import cx from "clsx";
 import { useId } from "@reach/auto-id";
-
-const paddings = {
-  xl: [1, 3],
-  lg: [2, 3],
-  md: [1, 2],
-  sm: ["px", 2],
-};
-
-const textSize = {
-  xl: TextSize.LG,
-  lg: TextSize.SM,
-  md: TextSize.SM,
-  sm: TextSize.XS,
-};
-
-const autosizeMinHeight = {
-  xl: 40,
-  lg: 40,
-  md: 32,
-  sm: 22,
-};
-
-const textColor = {
-  xl: Color.Neutral800,
-  lg: Color.Neutral700,
-  md: Color.Neutral700,
-  sm: Color.Neutral700,
-};
-
-const inputTypeFromMode = {
-  single: "text",
-  multi: "multiline",
-};
+import { useState } from "react";
 
 export function Input({
   id,
-  size = "md",
   icon,
   label,
   placeholder,
   hideLabel = false,
-  mode = "single",
-  autosize = autosizeMinHeight[size],
-  value,
   name,
-  innerRef,
-  enforceBorder = false,
+  onFocus = () => null,
+  onChange = () => null,
   ...props
 }) {
   const inputId = useId(id);
-  const [paddingY, paddingX] = paddings[size];
+  const [value, setValue] = useState(props.value);
+
+  function handleFocus(event) {
+    onFocus(event);
+  }
+
+  function handleChange(event) {
+    setValue(event.target.value);
+    onChange(event);
+  }
 
   return (
-    <Stack space={2} className="w-full">
+    <div className="flex flex-col items-stretch justify-start text-center space-y-2 w-full">
       {label && !hideLabel ? (
-        <Text
-          as="label"
-          color={Color.Neutral800}
-          weight={TextWeight.Medium}
-          size={TextSize.SM}
-          leading={TextLeading.Relaxed}
+        <label
+          className="text-neutral-800 font-medium text-sm leading-relaxed"
           htmlFor={inputId}
         >
           {label}
-        </Text>
+        </label>
       ) : null}
 
-      <Inline
-        space={2}
-        className={[
-          "border hover:border-neutral-200 focus-within:border-neutral-300",
-          "rounded-md bg-white w-full",
-          { "border-white": !enforceBorder },
-          { "border-neutral-200": enforceBorder },
-        ]}
-        p={[0, paddingX]}
+      <div
+        className={cx([
```

Comment:
just curious if this is neccessary here?

Reply:
Either way I think the package is a good addition

Reply:
yeah, there was a condition there before, then I forgot to modify this 😅